tests/network/probes.go

```diff
@@ -220,15 +220,11 @@ var _ = SIGDescribe("[ref_id:1182]Probes", func() {
 			table.Entry("[test_id:TODO]with working Exec probe", createExecProbe(period, initialSeconds, timeoutSeconds, "uname", "-a"), blankIPFamily, true),
 		)
 
-		table.DescribeTable("should fail the VMI", func(livenessProbe *v1.Probe, isExecProbe bool) {
+		table.DescribeTable("should fail the VMI", func(livenessProbe *v1.Probe) {
 			By("Specifying a VMI with a livenessProbe probe")
-			if isExecProbe {
-				vmi = tests.NewRandomFedoraVMIWithGuestAgent()
-				vmi.Spec.LivenessProbe = livenessProbe
-				vmi = tests.VMILauncherIgnoreWarnings(virtClient)(vmi)
-			} else {
-				vmi = createReadyCirrosVMIWithLivenessProbe(livenessProbe)
-			}
+			vmi = libvmi.NewFedora()
```

Comment:
Commit: `tests/network/probes: liveness: drop another isExecProbe bool arg `

This time, you have replace the Cirros with the Fedora guest.
If it is intentional, could you specify in the commit message why you think it is ok?

Reply:
it was intentional; but the code on the readiness probe lost it. commit message updated.

Reply:
One may argue that using Fedora is more costly in terms of resources & time.
But I am good with using Fedora for all network tests in general.